week-3/introduction_to_java_book_tasks/_11_1_Triangle_task/src/main/java/job_hacker/tasks/models/GeometricObject.java

```diff
@@ -0,0 +1,45 @@
+package job_hacker.tasks.models;
+
+import java.util.Date;
+
+public class GeometricObject {
+    String color;
+    boolean filled;
```

Comment:
for good naming you could rename this to isFilled

Reply:
Do you mean the getter? I called it like that by the way.
But that is not the naming convention for the field name itself, I guess.

Reply:
For naming convention it recommended to start the naming with "is" for boolean variables and getter the same. The setter without is